core/templates/components/summary-tile/exploration-summary-tile.component.html

```diff
@@ -4,7 +4,7 @@
                                 [activityId]="explorationId"
                                 [activityTitle]="explorationTitle"
                                 [isContainerNarrow]="isContainerNarrow"
-                                [activityActive]="explorationIsCurrentlyHoveredOver"
+                                activityActive="true"
                                 [isAddToPlaylistIconShown]="!isOwnedByCurrentUser">
   </oppia-learner-dashboard-icons>
```

Comment:
The `learner-dashboard-icons` component is used in the `collection-summary-tile` component as well. Just wondering if a change similar to this is needed there as well?

Reply:
And if so, maybe you could just have this property be initialized within the `learner-dashboard-icons` component, and not have it be passed down as input (if it's always going to be constant, i.e. `true`, anyway).

Reply:
@nithinrdy thanks for the suggestions, as we removed feature of hovering over `activityActive` input wasn't needed at all. Soo removed it from input as well as removed the related tests.